src/adv.py

```diff
@@ -36,16 +38,28 @@
 #
 # Main
 #
-
+print('Welcome to the adventure game.\n')
 # Make a new player object that is currently in the 'outside' room.
+player = Player(input('\nWhat is your name?\n'), room['outside'])
 
 # Write a loop that:
 #
 # * Prints the current room name
+
 # * Prints the current description (the textwrap module might be useful here).
+
 # * Waits for user input and decides what to do.
-#
+
 # If the user enters a cardinal direction, attempt to move to the room there.
 # Print an error message if the movement isn't allowed.
-#
+
 # If the user enters "q", quit the game.
+
+while True:
+    print('Press (c) to continue or (q) to quit\n')
```

Comment:
See if you can implement the quit command without the need for this prompt every time